scalafix-tests/unit/src/test/scala-2.12/scalafix/tests/cli/ScalafixImplSuite.scala

```diff
@@ -82,7 +82,7 @@ class ScalafixImplSuite extends FunSuite with DiffAssertions {
   }
 
   test("error") {
-    val cl = new URLClassLoader(Array())
+    val cl = new URLClassLoader(Array(), null)
```

Comment:
Ahhh, I tried the `fork := true` up above like you did, but didn't catch this one down here.

Reply:
It was also necessary to update `baseDirectory.in(Test) := baseDirectory.in(ThisBuild).value`. That should honestly be the default behavior in sbt, but changing the default would break a lot of builds